client/signup/steps/intent-screen/intents.jsx

```diff
@@ -0,0 +1,63 @@
+import { Card, Button } from '@automattic/components';
+import { localize } from 'i18n-calypso';
+import PropTypes from 'prop-types';
+import React from 'react';
+import buildingImage from 'calypso/assets/images/intent-screen/building.svg';
+import writingImage from 'calypso/assets/images/intent-screen/writing.svg';
+import './intents.scss';
+
+const useIntents = ( { translate, siteSlug } ) => {
+	return [
+		{
+			title: translate( 'Write' ),
+			description: translate( 'Share your ideas with the world' ),
+			illustration: writingImage,
+			href: `/page/${ siteSlug }/home`,
+			actionText: translate( 'Start writing' ),
```

Comment:
:information_source: String reuse speeds up translation and improves consistency. The following string might make a good alternative and has already been translated **3** times:
`translate( 'Start Writing' )` *ES Score: 11*
See 1 additional suggestions in the [PR translation status page](https://translate.wordpress.com/localci/status/automattic/wp-calypso/feat/intent-screen)